tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/CIEnvironmentVariableTests.cs

```diff
@@ -42,6 +43,22 @@ public static IEnumerable<object[]> GetJsonItems()
             }
         }
 
+        [InlineData("git@github.com:DataDog/dd-trace-dotnet.git", true)]
+        [InlineData("git@git@hub.com:DataDog/dd-trace-dotnet.git", false)]
+        [InlineData("git@git/hub.com:DataDog/dd-trace-dotnet.git", false)]
+        [InlineData("git@git$hub.com:DataDog/dd-trace-dotnet.git", false)]
+        [InlineData("github.com:DataDog/dd-trace-dotnet.git", false)]
+        [InlineData("https://github.com/DataDog/dd-trace-dotnet.git", true)]
+        [InlineData("https://github.com/DataDog/dd-trace-dotnet.dit", false)]
+        [InlineData("https://github.com/DataDog/dd-trace-dotnet.git123", false)]
+        [InlineData("git@gitlab.com:gitlab-org/gitlab.git", true)]
+        [InlineData("https://gitlab.com/gitlab-org/gitlab.git", true)]
```

Comment:
What about filepath URLs, e.g. `c:\repos\myrepo`. And yes, I've used these before, though they're pretty unlikely in CI, I grant you 😛

Reply:
Local development is not supported by CI Visibility. filepath URLs will be dropped by the backend, that's the reason the validation method doesn't include that type.